src/components/Main.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Product from './Product.jsx';
+
+export default function Main(props) {
+  const {products} = props;
```

Comment:
You can define your object directly in the component like this:
```suggestion
export default function Main({products}) {
```